src/jinja2/parser.py

```diff
@@ -57,9 +57,12 @@ def __init__(
         name: t.Optional[str] = None,
         filename: t.Optional[str] = None,
         state: t.Optional[str] = None,
+        preserve_comments: bool = False,
```

Comment:
suggestion: We could make it a keyword-only parameter for easier backward-compatibility maintenance in the future.